os/board/common/Kconfig

```diff
@@ -31,20 +31,20 @@ config FLASH_MINOR
 
 config FLASH_PART_SIZE
 	string "Flash partition size list (in KBytes)"
-	default "16,48,192,32,512,2400,1536,1536,1400,8,512,"
```

Comment:
Confirmed by testing on artik053. Modified values are enough for ssrw partition.